packages/mobile/src/send/saga.ts

```diff
@@ -154,12 +153,9 @@ function* sendPaymentOrInviteSaga({
       throw new Error("Can't send to recipient without valid e164 number or address")
     }
 
-    const ownAddress = yield select(currentAccountSelector)
-    const comment = features.USE_COMMENT_ENCRYPTION
-      ? yield call(encryptComment, reason, recipientAddress, ownAddress)
-      : reason
-
+    const ownAddress: string = yield select(currentAccountSelector)
     if (recipientAddress) {
+      const comment = yield call(encryptComment, reason, recipientAddress, ownAddress, true)
```

Comment:
Nit: consider renaming the result `encryptedComment`. As someone new to the codebase, the difference between `comment` and `reason` was already confusing to me.

Definitely optional but I think it would be useful if downstream references were also renamed `encryptedComment` so it is clear when we are dealing with a raw comment vs an encrypted one

Reply:
Will do!